### DISCUSSION TITLE
阿里ddns

### DISCUSSION THREAD

**Discussion by sunqin0310** [jeessy2/ddns-go#919] [Q&A]
阿里云的api不支持纯ipv6访问吗？我有个电脑，iPv4走的内网固定ip，无法修改ip那种，iPv4外网不通，但是ipv6可用，且公网，发现阿里云的ddns不可用，是不能纯ipv6使用吗？是这样吗？不过我发现porkbun.com的接口ipv6能用，

**Comment by jeessy2** [comment]
应该是

**Comment by sunqin0310** [comment]
好吧，希望国内这些服务商能早点支持 吧，就不用挂两家ns服务器了
